code/modules/projectiles/guns/projectile/constructable/blunderbuss.dm

```diff
@@ -97,7 +97,7 @@
 	for(var/i=1, i<=prohibited_items.len, i++)
```

Comment:
@ComicIronic, git-blame next time you want to know the author of a piece of code.